src/omnisharp/prioritization.ts

```diff
@@ -13,6 +13,8 @@ const priorityCommands = [
 ];
 
 const normalCommands = [
+    protocol.Requests.Completion,
+    protocol.Requests.CompletionResolve,
     protocol.Requests.AutoComplete,
```

Comment:
this makes sense to me, thanks. Also autocomplete is no longer used, so I guess we could clean up all the references to it @JoeRobich @333fred

Reply:
Looks like this is the only remaining reference, it and the constant can be deleted.

Reply:
@an-dr-eas-k would you mind remove this reference?